worker/pkg/benthos/sql/input_sql_raw.go

```diff
@@ -41,6 +39,7 @@ func RegisterPooledSqlRawInput(
 			if err != nil {
 				return nil, err
 			}
+			// return service.AutoRetryNacksToggled(conf, input)
```

Comment:
I added this here because this is what benthos has configured today. Although I'm leaving it commented out because we have never used this and it is supposed to retry downstream errors endlessly, which I don't think we really want our system to do today. Just leaving this here as a reminder if we ever want it.